src/components/Editor.js

```diff
@@ -292,9 +292,11 @@ const Editor = React.createClass({
     // happening ever again (in case CodeMirror re-applies the
     // class, etc).
     if (this.lastJumpLine) {
-      codeMirror.removeLineClass(
-        this.lastJumpLine - 1, "line", "highlight-line"
-      );
+      this.props.breakpoints.map(bp => {
+        codeMirror.removeLineClass(
+          bp.location.line - 1, "line", "highlight-line"
+        );
+      });
```

Comment:
hmm, we can navigate to any line by clicking on console link, or other features like "jump to original location"... so breakpoints is not enough

what if we just iterated over all of the code mirror lines and removed this class?